SimConnectMSFS/WasmModuleUpdater.cs

```diff
@@ -282,6 +282,20 @@ public bool DownloadHubHopPresets()
             return true;
         }
 
+        static public DateTime HubHopPresetTimestamp()
+        {
+            var lastModified = DateTime.MinValue;
+            try
+            {
+                lastModified = System.IO.File.GetLastWriteTimeUtc($@"{WasmEventsHubHopFolder}\{WasmEventsHubHopFileName}");
```

Comment:
I took the lastWriteTimeUtc because it is actually safer to go by the modified date than the create date.
Create date is set when copied. If a user copies an old preset file into the directory, it would have the current timestamp.

Not super relevant in normal circumstances.

Reply:
If it urges you to explain, do a code comment for future generations?

Reply:
it is in here.